instructor-dynamic.js

```diff
@@ -0,0 +1,104 @@
+/* eslint linebreak-style: ["error", "windows"] */
+/* Create Instructor Section Dynamically */
+
+const section = document.querySelector('#speakers');
+
+const InstructorsDataBase = [
+  {
+    image: 'images/sp-image1.png',
+    name: 'Yochai Benkler',
+    title: 'Berkman Professor of Computer Science at Harvard',
+    detail: 'Benkler studies commons based peer production, and published his seminal book The Wealth of Networks in 2006.',
+  },
+  {
+    image: 'images/sp-image2.png',
+    name: 'Sammy Berky',
+    title: 'CTO at Google Dev Academy',
+    detail: 'Sammy has a lot of experience about NLP technics and AI integration into Learning process.',
+  },
+  {
+    image: 'images/sp-image3.png',
+    name: 'Sammy Berky',
+    title: 'CTO at Google Dev Academy',
+    detail: 'Sammy has a lot of experience about NLP technics and AI integration into Learning process.',
+  },
+  {
+    image: 'images/sp-image4.png',
+    name: 'Sammy Berky',
+    title: 'CTO at Google Dev Academy',
+    detail: 'Sammy has a lot of experience about NLP technics and AI integration into Learning process.',
+  },
+  {
+    image: 'images/sp-image5.png',
+    name: 'Sammy Berky',
+    title: 'CTO at Google Dev Academy',
+    detail: 'Sammy has a lot of experience about NLP technics and AI integration into Learning process.',
+  },
+  {
+    image: 'images/sp-image6.png',
+    name: 'Sammy Berky',
+    title: 'CTO at Google Dev Academy',
+    detail: 'Sammy has a lot of experience about NLP technics and AI integration into Learning process.',
+  },
+];
+
+const loadInstructor = (data) => {
+  const headingCont = document.createElement('div');
+  headingCont.id = 'heading-container2';
+  section.appendChild(headingCont);
+
+  const sectionHeading = document.createElement('h2');
+  sectionHeading.className = 'lato dark section-heading';
+  sectionHeading.id = 'speakers-heading';
+  sectionHeading.innerText = 'Instructors';
+  headingCont.appendChild(sectionHeading);
+  const indicator = document.createElement('div');
+  indicator.id = 'indicator2';
+  indicator.className = 'indicator red-bg';
+  headingCont.appendChild(indicator);
+
+  const cardCont = document.createElement('div');
+  cardCont.id = 'speaker-cards-container';
+  section.appendChild(cardCont);
+
+  for (let i = 0; i < data.length; i += 1) {
+    const order = i + 1;
+    const instructorCard = document.createElement('div');
+    instructorCard.id = `speaker-card${order}`;
+    instructorCard.className = 'speaker-card';
```

Comment:
- [ ] As requested by the previous reviewer, Kindly ensure that your remove the border. This is to ensure that your design matches the original design and hence making your website professional ☺️ 

Your design 😎 
![image](https://github.com/s2jack/codewizards.github.io/assets/99394374/9232887c-c20c-4d86-bf89-70174a12a19f)

Original design 😎 
![249321065-1e6ddea8-e72f-465e-b83b-d76f16429271](https://github.com/s2jack/codewizards.github.io/assets/99394374/7ddab2b5-7d0b-459a-8977-bc9cc4e94c49)